pwm_frequency: 47.2
pwm_ms_bias: 0.09
servo_pwm_ms_bias: 0.05

thruster_ch_list: [1, 2, 3, 4]
thruster_topic_list: [thruster_1, thruster_2, thruster_3, thruster_4]
thruster_min_us: [1100, 1100, 1100, 1100]
thruster_max_us: [1900, 1900, 1900, 1900]
thruster_init_us: [1500, 1500, 1500, 1500]

led_ch_list: [5]
led_topic_list: [led_1]
led_min_us: [1100]
led_max_us: [1900]
led_init_us: [1100]

servo_ch_list: [6, 7]
servo_topic_list: [servo_1, servo_2]
servo_min_us: [500, 500]
servo_max_us: [2500, 2000]
servo_center_us: [1500, 1500]